### ISSUE TITLE
about run demo error

### ISSUE THREAD

**Issue by leizhu1989** [QwenLM/Qwen1.5#129]
os: ubuntu16.04
cuda: 118+torch2.0.1
transformer: 4.37.0

When I run code:

from transformers import AutoModelForCausalLM, AutoTokenizer
import torch
device = "cuda" # the device to load the model onto
model = AutoModelForCausalLM.from_pretrained(
    "/media/sdc/zhulei/Qwen1.5-14B-Chat",
    torch_dtype=torch.bfloat16,
    device_map="auto"
)
tokenizer = AutoTokenizer.from_pretrained("/media/sdc/zhulei/Qwen1.5-14B-Chat")
prompt = "Give me a short introduction to large language model."
messages = [
    {"role": "system", "content": "You are a helpful assistant."},
    {"role": "user", "content": prompt}
]
text = tokenizer.apply_chat_template(
    messages,
    tokenize=False,
    add_generation_prompt=True
)
model_inputs = tokenizer([text], return_tensors="pt").to(device)
generated_ids = model.generate(
    model_inputs.input_ids,
    max_new_tokens=512
)
generated_ids = [
    output_ids[len(input_ids):] for input_ids, output_ids in zip(model_inputs.input_ids, generated_ids)
]
response = tokenizer.batch_decode(generated_ids, skip_special_tokens=True)[0]

there has a error like this:
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation

**Comment by leizhu1989** [comment]
可以推理，这个只是打印信息而已

**Comment by jklj077** [comment]
duplicate of https://github.com/QwenLM/Qwen1.5/issues/26
